### PR TITLE
Set release and version labels in operator resources to v0.55.1

### PR DESCRIPTION
**Pull request by nikhil-thomas**
# Changes
- Repalace all version and release labels with 'v0.55.1'
- Add a script to replace verion labels during a release
- Add a script to replace all version and release related labels to the
    release version

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by concaf — APPROVED** [review]
awesome
/lgtm

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

File: cmd/kubernetes/operator/kodata/webhook/webhook.yaml [hunk 1]
```diff
@@ -86,8 +86,8 @@ metadata:
   name: tekton-operator-proxy-webhook
   namespace: tekton-pipelines
   labels:
-    version: "devel"
-    operator.tekton.dev/release: "devel"
+    version: v0.55.1
```
Comment (PuneetPunamiya) [inline cmd/kubernetes/operator/kodata/webhook/webhook.yaml:89]: just had a question, so with every release we run the script and get the version replace in all these the files right ?

Comment (PuneetPunamiya) [reply cmd/kubernetes/operator/kodata/webhook/webhook.yaml:89]: or is this for only release branch ?

Comment (PuneetPunamiya) [reply cmd/kubernetes/operator/kodata/webhook/webhook.yaml:89]: I see, this is only for release branch 😄

**Review by savitaashture — APPROVED** [review]
:tada:

**Pull request merged**